### PR TITLE
chore: update wevm project

### PR DESCRIPTION
**Pull request by tmm**
Updated Wevm details and added packages.

### CODE REVIEW

File: data/projects/w/wagmi-dev.yaml [hunk 1]
```diff
@@ -1,23 +1,30 @@
...
 websites:
   - url: https://wevm.dev
 social:
   twitter:
     - url: https://twitter.com/wevm_dev
 github:
-  - url: https://github.com/wagmi-dev
   - url: https://github.com/wevm
+  - url: https://github.com/wagmi-dev
 npm:
```
Comment (tmm) [inline data/projects/w/wagmi-dev.yaml:13]: Alphabetized these to make it easier to see what was missing. If order matters, happy to switch back and append new packages to the end.

File: data/projects/w/wagmi-dev.yaml
Comment (tmm) [inline data/projects/w/wagmi-dev.yaml:1]: If feasible, would be nice to switch from `wagmi-dev` to `wevm` everywhere. Happy to make those changes, just let me know!

Comment (ccerv1) [reply data/projects/w/wagmi-dev.yaml:1]: Hey Tom,
That would be great if you make the other changes too :)

- [ ] You need to change the project filename from `wagmi-dev.yaml` to `wevm.yaml` to match the `name` field in the YAML file. 
- [ ] You need to find-replace all the collection files that reference `wagmi-dev` as well. I see 6 from [this query](https://cloud.hasura.io/public/graphiql?endpoint=https%3A%2F%2Fopensource-observer.hasura.app%2Fv1%2Fgraphql&query=query+MyQuery+%7B%0A++projects_by_collection_v1%28where%3A+%7Bproject_name%3A+%7B_eq%3A+%22wagmi-dev%22%7D%7D%29+%7B%0A++++collection_name%0A++%7D%0A%7D%0A).

Soon we're going to have all the https://ecosyste.ms/ data fully onboarded. Will be very exciting to start creating downstream impact metrics for these packages! For example, getting all the onchain users / developers / gas fees for [viem](https://packages.ecosyste.ms/registries/npmjs.org/packages/viem) etc

**Review by ccerv1 — CHANGES_REQUESTED** [review]

**Review by ccerv1 — APPROVED** [review]
Looks good!

**Pull request merged**